src/HealthChecks.Aws.SystemsManager/DependencyInjection/SystemsHealthCheckBuilderExtensions.cs

```diff
@@ -0,0 +1,45 @@
+using HealthChecks.Aws.SystemsManager;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Extensions.DependencyInjection
```

Comment:
Please use file scoped namespaces for all new files.

Reply:
Fixed